packages/conf-openblas/conf-openblas.0.2.1/opam

```diff
@@ -31,6 +31,10 @@ depexts: [
   ["openblas"] {os = "macos" & os-distribution = "homebrew"}
   ["openblas"] {os = "freebsd"}
 ]
+x-ci-accept-failures: [
```

Comment:
Neat!

Reply:
Yes, I just merged this feature yesterday (https://github.com/ocurrent/opam-repo-ci/pull/62), sorry I forgot to tell you about it

Reply:
Nothing to be sorry about, don't worry. This is a fantastic news, I am glad we have it finally 🎉